pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -390,6 +395,17 @@ var supportedReinvocationPolicies = sets.NewString(
 	string(admissionregistration.IfNeededReinvocationPolicy),
 )
 
+var supportedValidationPolicyReason = sets.NewString(
+	string(metav1.StatusReasonUnauthorized),
+	string(metav1.StatusReasonForbidden),
+	string(metav1.StatusReasonNotFound),
+	string(metav1.StatusReasonInvalid),
+	string(metav1.StatusReasonTooManyRequests),
+	string(metav1.StatusReasonRequestEntityTooLarge),
+	string(metav1.StatusReasonConflict),
```

Comment:
there's a lot of clients that will auto-retry on conflict (including ours)... normally we only return this for uid and resourceVersion mismatches. Custom admission should likely not return this, so drop this one as well